sfepy/discrete/common/extmods/mesh.c

```diff
@@ -891,15 +891,15 @@ int32 mesh_get_centroids(Mesh *mesh, float64 *ccoors, int32 dim)
   MeshEntityIterator it0[1], it1[1];
 
   for (mei_init(it0, mesh, dim); mei_go(it0); mei_next(it0)) {
-    for (id = 0; id < nc; id++) {
+    for (id = 0; id < (uint32)nc; id++) {
```

Comment:
uint32 nc;

Reply:
nc - same as above comment...

Reply:
Also yes :) `mesh->geometry->dim` is uint32